frontends/web/src/components/rates/rates.tsx

```diff
@@ -141,13 +131,16 @@ function Conversion({
 
   const coin = amount.unit;
   let formattedValue = '---';
+  let toFormat = false;
+  let value = 0;
 
   if (amount.conversions) {
     if (amount.conversions[active] !== '')
       formattedValue = amount.conversions[active];
   } else {
     if (rates && rates[coin]) {
-      formattedValue = formatCurrency(rates[coin][active] * Number(amount.amount), active);
+      value = rates[coin][active] * Number(amount.amount);
```

Comment:
a lot of effort to make a format backend API endpoint, but this still uses floats :thinking: 

what is the plan here?

maybe use the `/coins/convertToFiat` endpoint in some way?

Reply:
As far as I understood, this is only used to convert to fiat the total coin balance in the account summary (https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/frontends/web/src/routes/account/summary/accountssummary.tsx#L262). I would suggest to directly add the `conversions` fiat values to the `total-balance` endpoint and remove this piece of code. I would do that in a separate PR, though, wdyt?